quiz.js

```diff
@@ -11,32 +11,40 @@ var inputFields = {
 var treeGrower = function() {
   inputFields.heightInput = parseInt(document.getElementById("height").value);
   inputFields.characterInput = document.getElementById("char").value;
-  treeChecker();
+  treeChecker(inputFields);
+  tree = [];
 }
 
 // This function contains an if,else statement that says if the height and character input fields are equal (===) to a blank field or nothing ("") then it will display an alert that says the fields must have value. Else, it will begin to run the for loop. The for loop will place an empty space in the front of the first character and decrease it with every loop. While the other will increase the character and decrease the space.
-var treeChecker = function () {
+var treeChecker = function (inputFields) {
   if (inputFields.heightInput === "" || inputFields.characterInput === "") {
```

Comment:
Right now, this does not accurately test if the fields are empty.  If I don't enter a value in the height field, the tree still generates a one-tall tree.  This is related to line 12, where you're running `parseInt()` on the input.  You're testing here for `inputFields.heightInput` to be empty, but `parseInt()` on nothing returns `NaN` which is not the empty string.